pctasks/core/pctasks/core/cosmos/containers/create_item_errors.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict, Optional, Type
+
+from pydantic import BaseModel
+
+from pctasks.core.cosmos.container import (
+    ContainerOperation,
+    CosmosDBContainer,
+    CosmosDBDatabase,
+    TriggerType,
+)
+from pctasks.core.cosmos.settings import CosmosDBSettings
+from pctasks.core.models.event import CreateItemErrorRecord
+
+T = CreateItemErrorRecord
+
+PARTITION_KEY = "/id"
+
+STORED_PROCEDURES: Dict[ContainerOperation, Dict[Type[BaseModel], str]] = {}
+
+TRIGGERS: Dict[ContainerOperation, Dict[TriggerType, str]] = {}
+
+
+class CreateItemErrorsContainer(CosmosDBContainer[T]):
```

Comment:
I had the same question, but wasn't sure how to answer it. But it sounds like thinking through the query pattern will help determine this.

> If instead we want to collect them by e.g. run_id, then we could store those documents in the same workflow_runs container rather than having separate containers.

We'll just have a single `run_id` for the lifetime of the workflow (at least until it's updated and re-submitted). So I'm not sure that partitioning by workflow ID is appropriate.

Reply:
These are merged into a single process-item-errors container now.